ext_tables.sql

```diff
@@ -27,34 +23,10 @@ CREATE TABLE tx_persons_domain_model_person (
 	status int(11) unsigned DEFAULT '0' NOT NULL ,
 	categories int(11) UNSIGNED DEFAULT '0' NOT NULL,
 
-	tstamp int(11) unsigned DEFAULT '0' NOT NULL,
```

Comment:
Why did you delete all the boilerplate fields?

Reply:
Since 9.3, it's no longer necessary to declare these fields in `ext_tables.sql` as the database schema analyzer automatically creates TYPO3 “management” related database columns out of the declared TCA. That's the reason why I removed all those fields.

https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.3/Feature-85160-AutoCreateManagementDBFieldsFromTCACtrl.html

Reply:
great